ironfish-cli/scripts/deploy-docker.sh

```diff
@@ -12,7 +12,18 @@ if [ -z "${PACKAGE_NAME-}" ]; then
     exit 1
 fi
 
+if [ -z "${GITHUB_SHA-}" ]; then
+    echo "Set GITHUB_SHA before running deploy-docker.sh"
+    exit 1
+fi
+
+if [ -z "${GITHUB_REF-}" ]; then
+    echo "Set GITHUB_REF before running deploy-docker.sh"
+    exit 1
+fi
 
 docker tag ironfish:latest ${REGISTRY_URL}/${PACKAGE_NAME}:latest
```

Comment:
Should we only use the `latest` tag if building from master? It seems like this will help distinguish images built on other branches, but it won't solve the problem of `latest` coming from a different branch.

Reply:
Still tagging it with latest atm so nothing unexpected breaks. After this merges I will go through terraform and migrate to using the master tag. Then we can remove latest. Good suggestion tho

Reply:
Sounds good to me!